### DISCUSSION TITLE
Can't connect to the server

### DISCUSSION THREAD

**Discussion by paulkczw** [Minestom/Minestom#1821] [Help] [answered]
Server Log: https://haste.suqatri.net/onitariwev.bash
Error Message: Server address too long: 1221
build.gradle: `implementation("com.github.Minestom.Minestom:Minestom:-SNAPSHOT")`
Client Version: 1.19.2

**Comment by LooFifteen** [comment, marked as answer]
If you are connecting via BungeeCord or Velocity, you'll need to specifically enable support for them in Minestom.
Check classes [BungeeCordProxy.java](https://github.com/Minestom/Minestom/blob/8ad2c7701fe059d35dfcaf86f013ac6d9dde5785/src/main/java/net/minestom/server/extras/bungee/BungeeCordProxy.java) and [VelocityProxy.java](https://github.com/Minestom/Minestom/blob/8ad2c7701fe059d35dfcaf86f013ac6d9dde5785/src/main/java/net/minestom/server/extras/velocity/VelocityProxy.java).

**Reply by paulkczw** [reply]
thx

**Comment by FrederikHeinrich** [comment]
> Server Log: https://haste.suqatri.net/onitariwev.bash build.gradle: `implementation("com.github.Minestom.Minestom:Minestom:-SNAPSHOT")` Client Version: 1.19.2

Jo first of April was already... you specify "1221" as server host IP...

**Reply by paulkczw** [reply]
My host address is not "1221". You can see that by the fact that the string length is printed in the error message and not the actual server address. https://github.com/Minestom/Minestom/blob/e5d0a43ba42c7023a93e8e33356909e614783ac7/src/main/java/net/minestom/server/network/packet/client/handshake/HandshakePacket.java#L46
